android/res/layout/view_ad_menuitem.xml

```diff
@@ -81,6 +74,7 @@
         android:layout_centerVertical="true"
         android:layout_marginRight="12dp"
         android:src="@drawable/ad_menu_speaker"
+        android:tint="@color/basic_white"
```

Comment:
if you tested in lower APIs( < 21), then I think you can safely remove it, because for what I know, this is doing nothing in lower APIs.